src/main/scala/uk/gov/nationalarchives/tdr/api/consignmentstatevalidation/ConsignmentStateTags.scala

```diff
@@ -20,3 +21,29 @@ object ValidateNoPreviousUploadForConsignment extends ConsignmentStateTag {
     }
   }
 }
+
+object ValidateFilesExistForMetadata extends ConsignmentStateTag {
```

Comment:
No, I suppose not. I've changed it so the sql integrity checks fail the inserts. I've refactored it a bit because if you let it just fail you get "Internal server error" in the response which is less than helpful.